src/plot/series/mark-series.js

```diff
@@ -78,7 +79,11 @@ class MarkSeries extends AbstractSeries {
             onMouseOver: e => this._valueMouseOverHandler(d, e),
             onMouseOut: e => this._valueMouseOutHandler(d, e)
           };
-          return <circle {...attrs} />;
+
+          if (typeof markType === 'function') {
```

Comment:
do you actually need a function? because then, you'd have to make sure the interaction handlers and all the stuff we may add in the future to scatterplot marks work.
my assumption is that your marks could all be paths. i mean even the defaut marks, circles, could be paths. and size can be dealt with with "transform: scale(x)". so the argument you'd pass would just be an SVG style path?